src/components/button/button.stories.tsx

```diff
@@ -40,7 +40,10 @@ const meta: Meta<typeof Button> = {
 export default meta
 
 export const Default = {
-  render: ({}) => <Button>Button</Button>,
+  args: {
+    children: 'Button',
+    'aria-label': 'Button aria label',
```

Comment:
**quibble:** I think this could potentially confuse engineers and make them think both need to be supplied 🤔 Without a value for `aria-label`, is that prop not appearing in the Story's controls?

Reply:
Will update this.